ddtrace/internal/datadog/profiling/dd_wrapper/include/constants.hpp

```diff
@@ -12,3 +13,12 @@ constexpr unsigned int g_backend_max_nframes = 512;
 
 // Maximum amount of time, in seconds, to wait for crashtracker send operations
 constexpr uint64_t g_crashtracker_timeout_secs = 5;
+
+// Name of the only runtime we support right now
+constexpr std::string_view g_runtime_name = "CPython";
```

Comment:
I think we do have some customers on pypy 🤔

Reply:
Wait, really?  Do cpython dynamic libraries even load on pypy?  If not, then crashtracker won't even begin to work there.

Reply:
oh, CPython extensions, probably not, Cython ones should though... so.... 🤷🏻 

also, TIL: https://pyo3.rs/v0.9.2/pypy.html

Reply:
Ah--I'll just do the right thing and propagate the interface for these things.

Reply:
`platform.python_implementation()` / `platform.python_version()` / etc are probably the best to use... just to be safe